docs/install.rst

```diff
@@ -73,6 +74,24 @@ Astropy also depends on other packages for optional features:
   of sigma-clipping and other functionality that may required computing statistics
   on arrays with NaN values.
 
+- `pytest-astropy <https://github.com/astropy/pytest-astropy>`_: See
+  :ref:`sourcebuildtest`.
+
+- `pytest-xdist <https://docs.pytest.org/en/3.0.0/xdist.html>`_: Used for
+  distributed testing.
+
+- `pytest-mpl <https://github.com/matplotlib/pytest-mpl>`_: Used for testing
+  with Matplotlib figures.
+
+- `IPython <https://ipython.org/>`_: Used for testing notebook interface of
```

Comment:
"duplicate target name" warning, add another underscore at the end to make the warning go away.

Reply:
Ops, thanks!